.github/workflows/ci.yml

```
@@ -0,0 +1,196 @@
+
+    - name: Perform CodeQL Analysis
+      uses: github/codeql-action/analyze@v3
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    needs: [test, security]
+    strategy:
+      matrix:
+        include:
+          - goos: linux
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)